scripts/sampler/__main__.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import annotations
+
+import argparse
+import asyncio
+import json
+import os
+import random
+from typing import Any
+
+from add_context import add_context
+from evaluate import evaluate_sample
+from finetune_gpt import generate_finetune_gpt
+from remove_context import remove_context
+from validate import validate_sample
+
+from mentat.sampler.sample import Sample
+from mentat.utils import mentat_dir_path
+from tests.benchmarks.benchmark_runner import (
```

Comment:
I couldn't run this until I added `tests` to `find_packages` in setup.py. Btw what do you think about moving the benchmark code out of the tests directory into its own top level directory benchmarking?

Reply:
Ah makes sense, I'll add that to setup.py.

Ya I like the idea of moving benchmarks to their own directory, mostly because I always forget the `--benchmark` flag.